build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-	id 'com.github.kt3k.coveralls' version '2.6.3'
+	id 'com.github.kt3k.coveralls' version '2.6.3' 
```

Comment:
Please let files unchanged if the changes have nothing to do with the main scope of the pull request.